ingest-app/src/cmr/ingest/api/subscriptions.clj

```diff
@@ -188,9 +218,14 @@
                             body
                             content-type
                             headers)
-          subscriber-id (get-subscriber-id new-subscription)]
+          subscriber-id (get-subscriber-id new-subscription)
+          token-user (api-core/get-user-id-from-token request-context)
```

Comment:
line 222 to 225 _can_ be rewritten with (as->) if your interested (I was pinged on this in one of my requests, so now I'm passing on the "favor").

Reply:
Very nifty!